src/main/java/frc/trigon/robot/subsystems/shooter/ShooterConstants.java

```diff
@@ -0,0 +1,48 @@
+package frc.trigon.robot.subsystems.shooter;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+
+public class ShooterConstants {
+    private static final int
+            LEFT_MOTOR_ID = 3,
+            RIGHT_MOTOR_ID = 4;
+
+    private static final boolean INVERTED = false;
+
+    private static final double VOLTAGE_SATURATION = 10;
```

Comment:
That is low. that means we are limiting out motors to 5000 RPM which is a bit of a waste

Reply:
I don't think we need more, if we will we will increase this.